lib/src/components/Avatar/docs/examples/texts.tsx

```diff
@@ -0,0 +1,16 @@
+import { Avatar } from '@/components/Avatar'
+
+const Example = () => {
+  return (
+    <>
+      <Avatar name="W" size="sm" />
+      <Avatar name="We" size="md" />
+      <Avatar name="Wel" size="lg" />
+      <Avatar name="Welc" size="xl" />
+      <Avatar name="Welco" size="xxl" />
+      <Avatar className="bg-brand-40 w-[130px] h-[130px] text-[60px]" name="Custom" size="lg" />
```

Comment:
```suggestion
      <Avatar className="bg-brand-40 size-[130px] text-[60px]" name="Custom" size="lg" />
```

Reply:
Don't you think it might be good to use `w-(--width-coming-from-theme-using-rem)` tailwind classes?